integration-tests/cookie-authentication/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You need to add this module to https://github.com/quarkiverse/quarkus-openapi-generator/blob/e622e6a450b7747751645371c5f5ac9404132776/integration-tests/pom.xml#L12

Reply:
Yes, you are right. Forgot to commit this change.